plugins/ril.c

```diff
@@ -267,19 +355,27 @@ void ril_post_sim(struct ofono_modem *modem)
 	if (mw)
 		ofono_message_waiting_register(mw);
 
-	ofono_call_forwarding_create(modem, rd->vendor, RILMODEM, rd->ril);
-
-	ofono_phonebook_create(modem, rd->vendor, RILMODEM, modem);
+	ofono_call_forwarding_create(modem, rd->vendor,
+			get_driver_type(rd, OFONO_ATOM_TYPE_CALL_FORWARDING),
+			rd->ril);
+	ofono_phonebook_create(modem, rd->vendor,
+			get_driver_type(rd, OFONO_ATOM_TYPE_PHONEBOOK), modem);
```

Comment:
Not sure now why I did this. However, in new version I have moved ofono_call_forwarding_create to post_online, in the same place as all SS services are created (call barring, call settings, etc.).